kgraphql/src/main/kotlin/com/apurebase/kgraphql/schema/structure/Type.kt

```diff
@@ -268,6 +269,28 @@ interface Type : __Type {
         override val possibleTypes: List<__Type>? = null
     }
 
+    class _ExecutionNode : Type {
```

Comment:
What is this class for, just wondering?

Reply:
Actually not sure myself :laughing: Just did the same as the previous maintainer did regarding the `Context`.

I'm currently waiting for the Kotlin 1.4 release. Where I will try to heavily rely on the new [`typeOf`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.reflect/type-of.html) functionality and try to simplify the internals of this library as much as possible.

Reply:
But as seen on `SchemaCompilation.kt:163` it just returns this instead of calculating anything about this `type`, which should also help with performance.